sematic/db/migrations/20221208183307_create_server_settings_file.py

```diff
@@ -1,32 +0,0 @@
-# Standard Library
```

Comment:
This migration file was already released in 0.21.0 and 0.21.1. It cannot be modified.
Please revert and update subsequent migration scripts.

Reply:
In principle I agree that we shouldn't rewrite the past and remove old migration.

In this case though, this migration creates the `server.yaml` file, only to remove it again in the subsequent migration, which seems a little overkill.

The new migration can deal with either situations (only settings.yaml is present, or both files are present), so this migration seems unnecessary.

I can bring it back if you really care about it, but it's unnecessary.

Reply:
There really are no consequences. These users will have an extra record in the `schema_versions` table but that has no effect.
If anything keeping the migration has a higher risk.
In any case, I'm not willing to die on that hill so I'll recover the migration.